src/apps/watchfaces/OswAppWatchfaceDigital.cpp

```diff
@@ -86,37 +95,24 @@ void drawTime() {
   char pm[] = "PM";
   OswHal* hal = OswHal::getInstance();
 
-  hal->gfx()->setTextSize(3);
+  hal->gfx()->setTextSize(OswConfigAllKeys::timeFormat.get() ? 4 : 3);
   hal->gfx()->setTextMiddleAligned();
   hal->gfx()->setTextLeftAligned();
-  hal->gfx()->setTextCursor(120 - hal->gfx()->getTextOfsetColumns(5.5), 120);
-
-  hal->getLocalTime(&hour, &minute, &second, &afterNoon);
-  timeOutput(hour, minute, second);
-  hal->gfx()->print(" ");
-  if (afterNoon) {
-    hal->gfx()->print(pm);
-  } else {
-    hal->gfx()->print(am);
+  hal->gfx()->setTextCursor(120 - hal->gfx()->getTextOfsetColumns(OswConfigAllKeys::timeFormat.get() ? 4 : 5.5),CoordY /*120*/);
```

Comment:
Small conditional statements occur in the function while merging two `drawTime24Hour` and `drawTime` periods. Can we solve this problem?

Reply:
I think you can merge them. Hey, our core runs on 8 or 16 Mhz, so one small `if` does not kill you.